questionnaire-ko.json

```diff
@@ -1,434 +1,434 @@
-          "type": "YES_NO",
+            "4.1",
+            "4.1.1"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "식별된 라이선스에서 요구하는대로 컴플라이언스 결과물을 공급 대상 소프트웨어와 함께 배포하기 위한 프로세스를 설명하는 문서화된 절차가 있습니까?",
+          "number": "4.b",
+          "type": "YES_NO",
           "specReference": [
-            "4.1",
-            "4.1.2"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Part 5",
-      "title": "5. Understanding open source community engagements",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a policy for contribution to open source projects on behalf of the organization?",
-          "number": "5.a",
-          "type": "YES_NO",
+            "4.1",
+            "4.1.2"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "공급 대상 소프트웨어의 컴플라이언스 결과물 사본을 보관합니까?",
+          "number": "4.c",
+          "type": "YES_NO",
           "specReference": [
+            "4.1",
+            "4.1.2"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Yes",
+      "title": "5. 4.c",
+      "questions": [
+        {
+          "correctAnswer": "네",
+          "question": "Part 5",
+          "number": "5.a",
+          "type": "YES_NO",
+          "specReference": [
             "5.1",
-            "5.1.1"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure governing open source contributions?",
-          "number": "5.b",
-          "type": "YES_NO",
+            "5.1.1"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "조직을 대신하여 오픈소스 프로젝트에 기여하는 것을 관리하는 정책이 있습니까?",
+          "number": "5.b",
+          "type": "YES_NO",
           "specReference": [
-            "5.1",
-            "5.1.2"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for making all Software Staff aware of the open source contribution policy?",
-          "number": "5.c",
-          "type": "YES_NO",
+            "5.1",
+            "5.1.2"
+          ]
+        },
+        {
+          "correctAnswer": "네",
+          "question": "오픈소스 기여를 관리하는 문서화된 절차가 있습니까?",
+          "number": "5.c",
+          "type": "YES_NO",
           "specReference": [
-            "5.1",
-            "5.1.3"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Part 6",
-      "title": "6. Adherence to the specification requirements",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have documentation confirming that your Program meets all the requirements of this specification?",
-          "number": "6.a",
-          "type": "YES_NO",
+            "5.1",
+            "5.1.3"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Yes",
+      "title": "6. 5.c",

```

Comment:
This is incorrect - it should be a translation of the title

Reply:
PR later.